components/style/properties/longhand/inherited_svg.mako.rs

```diff
@@ -112,17 +112,17 @@ ${helpers.single_keyword("clip-rule", "nonzero evenodd",
                          animation_value_type="discrete",
                          spec="https://www.w3.org/TR/SVG11/masking.html#ClipRuleProperty")}
 
-${helpers.predefined_type("marker-start", "UrlOrNone", "Either::Second(None_)",
+${helpers.predefined_type("marker-start", "url::UrlOrNone", "computed_value::T::none()",
                           products="gecko",
                           animation_value_type="discrete",
                           spec="https://www.w3.org/TR/SVG2/painting.html#VertexMarkerProperties")}
 
-${helpers.predefined_type("marker-mid", "UrlOrNone", "Either::Second(None_)",
+${helpers.predefined_type("marker-mid", "url::UrlOrNone", "computed_value::T::none()",
                           products="gecko",
                           animation_value_type="discrete",
                           spec="https://www.w3.org/TR/SVG2/painting.html#VertexMarkerProperties")}
 
-${helpers.predefined_type("marker-end", "UrlOrNone", "Either::Second(None_)",
+${helpers.predefined_type("marker-end", "url::UrlOrNone", "computed_value::T::none()",
```

Comment:
nit: I'd do `computed::url::UrlOrNone::none()` here and in the rest.

Reply:
I'm following your advice blindly on this one. To figure out `computed_value::T::blablabla` works I had to check out the helper and see what's available. I understand why `computed::url::UrlOrNone::none()` but why is it better?

Reply:
I guess it's clearer because it shows the path to the file instead of referring to the (a bit more obscure) helper?